physical_properties/pKa/experimental_data/README.md

```diff
@@ -0,0 +1,14 @@
+## Experimental pKa values of SAMPL6 pKa challenge molecules
+This directory contains results of the pKa measurements performed with Sirius T3 instruments for 24 small molecules. 
+3 independent replicate pKa measurements were done for each molecule, starting from the same DMSO stock solution.
+Please see [SAMPL6/pKa Challenge Instructions/Experimental Details](https://github.com/MobleyLab/SAMPL6/blob/master/pKa_challenge_instructions.md#experimental-details) for more details about the experiment.
+
+`pKa_experimental_values.csv` file contains mean and standard error of the mean(SEM) of experimental macroscopic pKa values, calculated using `calc_pKa_value_statistics.py` script.
+Since pKa values measured by Sirius T3 were reported with 2 decimal places, we have reported SEM as 0.01 in cases where SEM values calculated from 3 replicates were lower than 0.01.  
+
+## Manifest
+- `pKa_experimental_values.csv` - Mean and SEM of pKa values organized in a table. Up to three pKa values were reported based on how many pKas were measured in the experimental interval of 2-12.
+- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values.
+- `pKa_replicate_experimental_results.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory).
+- `experiment_reports\` - This directory contains reports generated by Sirius T3 software for the results of pKa measurements. Refer to `pKa_replicate_experimental_results.csv` for matching SAMPL6 molecule IDs to names of report files.

```

Comment:
This seems like A LOT of work just to find the correct eperimental report to read. Could you rename the `experiment_reports/` files to include the SAMPL6 molecule ID? 
(as a side note the '\' in this line is the wrong one for directory separation, should be '/'

Reply:
I added SAMPL6 molecule IDs as prefixes to report files.

Reply:
perfect!